app/_components/DataBox/DataboxMed.tsx

```
@@ -8,37 +8,50 @@ type Session = {
+    return 'text-blue-950';
+  };
   return (
     <div
       className='databox-med
     '
     >
       <h2 className='databox__title'>
-        Student's grade{' '}
-        <span className='text-lime-600 font-bold'>{gradeChangeWord}</span> since last
-        exam
+        Student's grade <span className={gradeColor()}>{gradeChangeWord}</span>{' '}
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)